error/src/lib.rs

```diff
@@ -464,6 +464,9 @@ impl CaliptraError {
     pub const RUNTIME_GET_FMC_CSR_UNSUPPORTED_FMC: CaliptraError =
         CaliptraError::new_const(0x000E0055);
 
+    pub const RUNTIME_REVOKE_EXPORTED_CDI_HANDLE_FAILED: CaliptraError =
```

Comment:
nit: Maybe rename to `RUNTIME_REVOKE_EXPORTED_CDI_HANDLE_NOT_FOUND`

Reply:
Will do